### ISSUE TITLE
should the readme.md be updated

### ISSUE THREAD

**Issue by axs89** [FunnelEnvy/optimizely-cli#61]
The Known Issues and Release History aren't really accurate anymore. Should they be removed? Should the Release History migrate to a roadmap doc?

**Comment by nselvidge** [comment]
Yes they definitely should. I think removing these and adding in some info explaining what you can put in the experiment/variation json files (as pointed out in #47 ) would make sense.
